islandora_solr.api.php

```diff
@@ -190,6 +190,15 @@ function hook_islandora_solr_search_rss_item_alter($item, $doc) {
   $item['description'] = 'this is the new rss item description';
 }
 
+/**
+ * Implements hook_islandora_solr_facet_variables_alter().
+ */
+function hook_islandora_solr_facet_variables_alter(&$buckets) {
+  foreach ($buckets AS $bucket) {
+    $bucket['link'] = $new_link;
```

Comment:
Not super important, but I think again you are altering a copy of each element in `$buckets` and not the actual `$buckets` array. So I don't think this example will change anything. Probably need 
```
foreach ($buckets AS &$bucket) {
```
to alter the original array.

Reply:
foreach ($buckets as &$bucket) could do the trick. With emphasis on the lower case as... DCS.